packages/contracts/ts/deploy.ts

```diff
@@ -197,71 +221,76 @@ export const deployVerifier = async (signer?: Signer, quiet = false): Promise<Ve
  */
 export const deployConstantInitialVoiceCreditProxy = async (
   args: { amount: number },
+  proxyFactory: ConstantInitialVoiceCreditProxyFactoryContract,
   signer?: Signer,
-  proxyFactory?: ConstantInitialVoiceCreditProxyFactoryContract,
-  quiet = false,
-): Promise<[ConstantInitialVoiceCreditProxy, ConstantInitialVoiceCreditProxyFactoryContract]> => {
+): Promise<ConstantInitialVoiceCreditProxy> => {
   if (!signer) {
     throw new Error("Signer is not provided");
   }
 
-  const initialVoiceCreditProxyFactory =
-    proxyFactory ??
-    (await deployContract<ConstantInitialVoiceCreditProxyFactoryContract>(
-      EInitialVoiceCreditProxiesFactories.Constant,
-      signer,
-      quiet,
-    ));
-
   const initialVoiceCreditProxy = await deployProxyClone<ConstantInitialVoiceCreditProxy, number[]>({
     factory: new ConstantInitialVoiceCreditProxyFactory(signer),
-    proxyFactory:
-      initialVoiceCreditProxyFactory as unknown as IFactoryLike<ConstantInitialVoiceCreditProxyFactoryContract>,
+    proxyFactory: proxyFactory as unknown as IFactoryLike<ConstantInitialVoiceCreditProxyFactoryContract>,
     args: [args.amount],
     signer,
   });
 
-  return [initialVoiceCreditProxy, initialVoiceCreditProxyFactory];
+  return initialVoiceCreditProxy;
 };
 
 /**
- * Deploy a ERC20VotesInitialVoiceCreditProxy contract
- * @param args - the deploy ERC20VotesInitialVoiceCreditProxy arguments
+ * Deploy a ERC20VotesInitialVoiceCreditProxyFactory contract
  * @param signer - the signer to use to deploy the contract
  * @param proxyFactory - the optional proxy factory to reuse for deployment
  * @param quiet - whether to suppress console output
- * @returns the deployed ERC20VotesInitialVoiceCreditProxy contract
+ * @returns the deployed ERC20VotesInitialVoiceCreditProxyFactory contract
  */
-export const deployERC20VotesInitialVoiceCreditProxy = async (
-  args: { token: string; snapshotBlock: bigint; factor: bigint },
+export const deployERC20VotesInitialVoiceCreditProxyFactory = async (
   signer?: Signer,
-  proxyFactory?: ERC20VotesInitialVoiceCreditProxyFactoryContract,
   quiet = false,
-): Promise<[ERC20VotesInitialVoiceCreditProxy, ERC20VotesInitialVoiceCreditProxyFactoryContract]> => {
+): Promise<ERC20VotesInitialVoiceCreditProxyFactoryContract> => {
   if (!signer) {
     throw new Error("Signer is not provided");
   }
 
   const erc20VotesInitialVoiceCreditProxyFactory =
-    proxyFactory ??
-    (await deployContract<ERC20VotesInitialVoiceCreditProxyFactoryContract>(
+    await deployContract<ERC20VotesInitialVoiceCreditProxyFactoryContract>(
```

Comment:
You can return here without adding const

Reply:
Done. When inspecting the `no-return-await` rule I discovered it is now [no longer recommend by eslint](https://eslint.org/docs/latest/rules/no-return-await), so I turned it off in `eslintrc.js`